src/models/db.js

```diff
@@ -28,6 +28,19 @@ const IDLE           = 0
 const INITIALIZING   = 1;
 const READY          = 2;
 
+const hooks = [
+  'beforeValidate',
+  'afterValidate',
+  'beforeCreate',
+  'afterCreate',
+  'beforeDestroy',
+  'afterDestroy',
+  'beforeRestore',
+  'afterRestore',
+  'beforeUpdate',
+  'afterUpdate'
+];
```

Comment:
Maybe add these as constants in `constants.js`, so you can use them from the tests as well.